src/main/java/io/confluent/connect/elasticsearch/ElasticsearchSinkConnectorConfig.java

```diff
@@ -70,6 +70,11 @@
 
   public static final String TYPE_NAME_CONFIG = "type.name";
   private static final String TYPE_NAME_DOC = "The Elasticsearch type name to use when indexing.";
+
+  /**
```

Comment:
I believe we don't use javadoc style comments with config variables. Correct me if I'm wrong.

Reply:
This is mostly for reference. I suppose we should simply use `//` here

Reply:
Indeed, we use `//` everywhere besides javadoc (and copyright header)

Reply:
An addition to my previous comment: The `@deprecated` annotation applies to fields as well (https://docs.oracle.com/javase/tutorial/java/annotations/predefined.html). Also, if we use Javadoc, the java compiler can report a warning when we use these fields in the code.

Reply:
My point was that we skip javadocs for all the configs, it wouldn't make much sense to add it only for the deprecated ones. 

If you want to include the warning during build, feel free to add the `@Deprecated` annotation to the variable (outside a comment - javadoc or regular).

Reply:
We should use the `@Deprecated` annotation at least, and optionally the JavaDoc tag. In this case, I think the annotation would be sufficient.

Reply:
Got it. Removed the javadoc, and replaced it with a `@Deprecated` annotation.